apps/i18n/common/en_us.json

```diff
@@ -1780,6 +1780,12 @@
   "verifyStudentCount": "I taught Units 1, 2, and 3 of Code.org's CS Discoveries curriculum to a classroom of at least 10 students.",
   "verifyYear": "I plan on teaching Unit 6 (Physical Computing) of CS Discoveries to a classroom of at least 10 students by the end of the '20-'21 school year.",
   "version": "Version",
+  "versionHistory_ajaxFailure": "An error occurred.",
+  "versionHistory_clearProgress_prompt": "Are you sure you want to clear all progress for this level&#63;",
```

Comment:
I'd suggest just using an ASCII question mark; I'm not certain why the original didn't, but we definitely don't want translators to have to deal with this

Reply:
fixed